packages/components/input/custom-elements.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Ehm, `input` doesn't exist anymore right? That's now `text-input`? So this file can be deleted?